### PR TITLE
adding the jdm classics task to library

### PR DESCRIPTION
**Pull request by JuneOh710**

### CODE REVIEW

**Comment by vsoch** [comment]
@JuneOh710 just rebase with master and this should build!

**Comment by JuneOh710** [comment]
@vsoch Did I do the rebase correctly?

**Comment by vsoch** [comment]
Hmm I don't think so - let me see if I can help.

File: docs/_library/jdm-classics.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+layout: experiment
+name:  "jdm-classics"
+maintainer: "@vsoch"
```
Comment (vsoch) [inline docs/_library/jdm-classics.md:4]: Do you want to put your GitHub alias here?

Comment (JuneOh710) [reply docs/_library/jdm-classics.md:4]: Yes, that works for me.

Comment (vsoch) [reply docs/_library/jdm-classics.md:4]: Great! Could you please update the PR for this experiment file and the previously merged one?

**Comment by vsoch** [comment]
I think this one was merged in another PR.

**Pull request closed without merging**